example/lib/main.dart

```diff
@@ -11,21 +11,19 @@ class MyApp extends StatefulWidget {
 }
 
 class _MyAppState extends State<MyApp> {
-  ESewaPnp _esewaPnp;
-  ESewaConfiguration _configuration;
+  ESewaPnp? _esewaPnp;
+  ESewaConfiguration? _configuration = ESewaConfiguration(
+    clientID: "JB0BBQ4aD0UqIThFJwAKBgAXEUkEGQUBBAwdOgABHD4DChwUAB0R",
+    secretKey: "BhwIWQQADhIYSxILExMcAgFXFhcOBwAKBgAXEQ==",
+    environment: ESewaConfiguration.ENVIRONMENT_TEST,
+  );
```

Comment:
If you are already assigning `_configuration`, what's the point of making it `nullable`?

Revert it to old code, and then make it `late` initializer.